### PR TITLE
Personalsettings - fix marginLeft as it breaks the layout

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

| Before | After |
|---|---|
| 192 168 146 130_8443_index php_settings_user | 192 168 146 130_8443_index php_settings_user (1) |

<details>
<summary>For my own testing</summary>

```
docker run -it \
-e SERVER_BRANCH=fix/noid/fix-personal-settings \
-p 8443:443 \
-e TRUSTED_DOMAIN=192.168.146.130 \
--name nextcloud-easy-test \
ghcr.io/szaimen/nextcloud-easy-test:latest
```

</details>

### CODE REVIEW

File: apps/settings/src/components/PersonalInfo/ProfileVisibilitySection/ProfileVisibilitySection.vue [hunk 1]
```diff
@@ -83,7 +85,7 @@ export default {
 		subscribe('settings:profile-enabled:updated', this.handleProfileEnabledUpdate)
 		// TODO remove this when not used once the settings layout is updated
 		window.onresize = () => {
-			this.marginLeft = window.matchMedia('(min-width: 1200px)').matches
+			this.marginLeft = window.matchMedia('(min-width: 1750px)').matches
```
Comment (szaimen) [inline apps/settings/src/components/PersonalInfo/ProfileVisibilitySection/ProfileVisibilitySection.vue:88]: @Pytal 1600px should in theory also work but 1200px is much too less in my testing...
Shall I change it to 1600px or are you fine with 1750px?

Comment (szaimen) [reply apps/settings/src/components/PersonalInfo/ProfileVisibilitySection/ProfileVisibilitySection.vue:88]: Ping @Pytal :)

Comment (Pytal) [reply apps/settings/src/components/PersonalInfo/ProfileVisibilitySection/ProfileVisibilitySection.vue:88]: Would go with 1600px :)

Comment (szaimen) [reply apps/settings/src/components/PersonalInfo/ProfileVisibilitySection/ProfileVisibilitySection.vue]: done :)

**Comment by szaimen** [comment]
Should be ready to go now :)

**Review by artonge — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**